packages/management/addons-manager-v2/bundle/config/upstream/cpi.tanzu.vmware.com_cpiconfigs.yaml

```diff
@@ -0,0 +1,237 @@
+
```

Comment:
question: will `make manifest` automatically update this file ? Or we have to manually sync the updates whenever there is a API change.

Reply:
No, vendir sync needs to be done after picking which commit to target.